### ISSUE TITLE
Error: Error: z error - expecting three columns;

### ISSUE THREAD

**Issue by achubaty** [PredictiveEcology/LandR#91]
e.g., [GHA logs](https://github.com/PredictiveEcology/LandR/actions) on Windows show failures:

```r
> firePerimeters <- Cache(prepInputsFireYear,
+                         url = paste0("https://cwfis.cfs.nrcan.gc.ca/downloads",
+                         "/nfdb/fire_poly/current_version/NFDB_poly.zip"),
+                         fun = "sf::st_read",
+                         destinationPath = tempDir,
+                         rasterToMatch = ras2match)

<snip>

Reading layer `NFDB_poly_20210707' from data source 
  `C:\Users\runneradmin\AppData\Local\Temp\Rtmp6hRlg1\working_dir\RtmpoxCNVb\NFDB_poly_20210707.shp' 
  using driver `ESRI Shapefile'
Error: Error: z error - expecting three columns;
```

@ianmseddy what was the solution for these erros you had fixed previously elsewhere?

**Comment by CeresBarros** [comment]
try removing `fun = "sf::st_read"`

**Comment by achubaty** [comment]
Per https://github.com/r-spatial/sf/issues/1592, this is fixed in the development version of `sf` (v1.0-10):

```r
install.packages("sf", repos = "https://r-spatial.r-universe.dev") ## prebuilt binaries
## or
remotes::install_github("r-spatial/sf") ## needs compilation
```
